src/w3c/headers.js

```diff
@@ -301,6 +301,15 @@ export async function run(conf) {
     const hint = docLink`Please set ${"[license]"} to "w3c-software-doc" instead.`;
     showError(msg, name, { hint });
   }
+  if (!licenses.has(conf.license)) {
+    const msg = `The license "\`${conf.license}\`" is not supported.`;
```

Comment:
Not recognized? No strong opinion though.

Reply:
Will keep it as is for now.